elftools/elf/descriptions.py

```diff
@@ -463,30 +463,28 @@ def describe_attr_tag_arm(tag, val, extra):
     ELF_NOTE_OS_SYLLABLE='Syllable',
 )
 
-
-_DESCR_RELOC_TYPE_i386 = dict(
-        (v, k) for k, v in iteritems(ENUM_RELOC_TYPE_i386))
-
-
-_DESCR_RELOC_TYPE_x64 = dict(
-        (v, k) for k, v in iteritems(ENUM_RELOC_TYPE_x64))
-
-
-_DESCR_RELOC_TYPE_ARM = dict(
-        (v, k) for k, v in iteritems(ENUM_RELOC_TYPE_ARM))
-
-
-_DESCR_RELOC_TYPE_AARCH64 = dict(
-        (v, k) for k, v in iteritems(ENUM_RELOC_TYPE_AARCH64))
-
-
-_DESCR_RELOC_TYPE_MIPS = dict(
-        (v, k) for k, v in iteritems(ENUM_RELOC_TYPE_MIPS))
-
-
-_DESCR_D_TAG = dict(
-        (v, k) for k, v in iteritems(ENUM_D_TAG))
-
+def _reverse_dict(d, low_priority=()):
+    out = {}
+    for k, v in iteritems(d):
+        if v in out and k in low_priority:
+            continue
+        out[v] = k
+    return out
+
+_DESCR_RELOC_TYPE_i386 = _reverse_dict(ENUM_RELOC_TYPE_i386)
+_DESCR_RELOC_TYPE_x64 = _reverse_dict(ENUM_RELOC_TYPE_x64)
+_DESCR_RELOC_TYPE_ARM = _reverse_dict(ENUM_RELOC_TYPE_ARM)
+_DESCR_RELOC_TYPE_AARCH64 = _reverse_dict(ENUM_RELOC_TYPE_AARCH64)
+_DESCR_RELOC_TYPE_MIPS = _reverse_dict(ENUM_RELOC_TYPE_MIPS)
+_DESCR_D_TAG = _reverse_dict(ENUM_D_TAG, (
+    # these are 'meta-tags' marking semantics of numeric ranges of the enum
```

Comment:
Can you link to the relevant documentation pages here (as you did in the issue)?